.travis/build-deps.sh

```diff
@@ -63,23 +76,41 @@ build_openssl () {
 }
 
 # Enable ccache
-if [ "${TRAVIS_OS_NAME}" != "osx" ]; then
+if [ "${TRAVIS_OS_NAME}" != "osx" ] && [ -z "${CHOST+xxx}" ]; then
     # ccache not available on osx, see:
     # https://github.com/travis-ci/travis-ci/issues/5567
     mkdir -p "${HOME}/bin"
     ln -s "$(which ccache)" "${HOME}/bin/${CC}"
     PATH="${HOME}/bin:${PATH}"
 fi
 
-# Download and build crypto lib
-mkdir -p download-cache
-if [ "${SSLLIB}" = "openssl" ]; then
-    download_openssl
-    build_openssl
-elif [ "${SSLLIB}" = "mbedtls" ]; then
-    download_mbedtls
-    build_mbedtls
+if [ ! -z ${CHOST+xxx} ]; then
+      # cross build
+      echo "deb http://archive.ubuntu.com/ubuntu xenial main universe" | sudo tee -a /etc/apt/sources.list.d/xenial.list
+      echo "deb http://archive.ubuntu.com/ubuntu xenial main" | sudo tee -a /etc/apt/sources.list.d/xenial.list
```

Comment:
openvpn master branch uses firewall manipulation (for dns filtering purposes), it does not compiled on trusty